app/javascript/components/ProductEdit/ContentTab/PageTab.tsx

```diff
@@ -66,10 +67,31 @@ export const PageTab = ({
     "outline-key": "Page has license key",
   };
   return (
-    <div role="tab" onClick={onClick} aria-selected={selected}>
-      {!disabled ? <div aria-grabbed={dragging} /> : null}
+    <PageListItem
+      onClick={onClick}
+      isSelected={selected}
+      // .sortable-* are created by react-sortablejs, and we can't add Tailwind classes to them directly.
+      className="group/tab relative [&_.sortable-drag]:border [&_.sortable-drag]:bg-muted [&.sortable-ghost]:outline [&.sortable-ghost]:outline-accent [&.sortable-ghost]:outline-dashed [&.sortable-ghost>_*]:opacity-30"
+      role="tab"
+    >
+      {!disabled ? (
+        <Icon
+          name="outline-drag"
+          className="invisible absolute left-0 text-muted-foreground group-hover/tab:visible"
+          aria-grabbed={dragging}
+        />
+      ) : null}
       <Icon name={icon} aria-label={iconLabels[icon]} />
-      <span className="content">{renaming ? <EditorContent editor={editor} /> : titleWithFallback(page.title)}</span>
+      <span className="flex-1">
+        {renaming ? (
+          <>
+            <EditorContent editor={editor} className="cursor-text" />
+            <span className="absolute inset-0 outline-2 -outline-offset-2 outline-accent" />
```

Comment:
Can we just replace the wrapper fragment with this and not have to use `absolute` here?

Reply:
True, we can just add the styles on the PageListItem